chart/values.yaml

```diff
@@ -220,6 +220,14 @@ kube-prometheus-stack:
   ## Deploy a Grafana instance
   ##
   grafana:
+
+    ## Security patch CVE-2025-4123 https://grafana.com/security/security-advisories/cve-2025-4123/
+    ##
+    image:
+      registry: docker.io
+      repository: grafana/grafana
+      tag: "10.4.18-security-01"
```

Comment:
I would prefer latest 10.4.19-security-01.
We will get on top of CVE-2025-4123, also CVE-2025-3580 and CVE-2025-3415, see https://grafana.com/blog/2025/05/23/grafana-security-release-medium-and-high-severity-security-fixes-for-cve-2025-4123-and-cve-2025-3580/ and https://grafana.com/blog/2025/06/13/grafana-security-update-medium-severity-security-release-for-cve-2025-3415/